src/lib/util/useViewportSize.ts

```diff
@@ -1,34 +1,32 @@
-import { useCallback, useEffect, useState } from 'react'
+import { useState } from 'react'
+import { useIsomorphicLayoutEffect } from '../../components/shared/useIsomorphicLayoutEffect'
 
-const getBreakpointSize = (width: number) => (width <= 512 ? 'small' : 'big')
 
 /**
  * Determine the viewport size depending on the inner sizing of the window, useful
  * for handling layout choices depending on the size of the browser
  */
 export const useWindowSize = () => {
```

Comment:
The changes here fix hydration errors and make server-side only renderings use 1024x768 as fallback window size.